rage/Cargo.toml

```diff
@@ -66,11 +66,11 @@ pinentry = "0.5"
 rust-embed = "6"
 
 # rage-mount dependencies
-fuse_mt = { version = "0.5.1", optional = true }
+fuse_mt = { git = "https://github.com/wfraser/fuse-mt.git", rev = "72be0d4fee79c4852a50474c9724934cf465aaa2", optional = true }
```

Comment:
Sadly I cannot merge this PR, because Rust crates that have unpublished dependencies cannot themselves be published. As soon as there is a published release of `fuse-mt` that includes the change to `fuser`, we can migrate to it.

Reply:
Thanks for the feedback. I made a request on `fuse-mt` repo https://github.com/wfraser/fuse-mt/issues/40. Until `fuse-mt` makes a new release is there anything I need to fix on this PR?

Reply:
At a glance the rest of the PR looks like what I'd expect for this change.

Reply:
`fuse-mt` released a new version 0.6.0. I've updated the PR. Can you review again?